flex1.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flex</title>
+    <link rel="stylesheet" href="styleflex.css" />
+</head>
+<body>
+    <div class="container">
+        <div class="pricing-table">
+            
+            <div class="pricing-plan">
+                <img src="img/plane1.png" alt="paperplane" class="pricing-img">
```

Comment:
The images are missing from the PR. Please add them in the PR as well.

Reply:
![Screenshot (32)](https://user-images.githubusercontent.com/89713248/133191631-a7a75ed0-7faa-4c47-8cdc-f8cf8371f32f.png)

Reply:
![Screenshot (33)](https://user-images.githubusercontent.com/89713248/133191683-27dda952-75bb-4336-ba8d-88f4ae5d4fc2.png)